### ISSUE TITLE
Aggregator Editor: Cannot close aggregator/formatter editor in schema config

### ISSUE THREAD

**Issue by chanulee1** [specify/specify7#3146]
Closing the editor does not go back to the schema config.

https://user-images.githubusercontent.com/103299204/224397598-25026a07-afa8-421b-9a6c-2f1561b6a37d.mp4

**Comment by maxxxxxdlp** [comment]
Yeah, I mentioned in the pull request description that the `+` and pencil buttons won't work properly just yet:

>   - Note, for now, you can only open the editor in a new tab (right-click on the `+` or pencil and "Open in new tab"). Once `agent-merging` is merged (it has some code changes that are necessary here), then you can just click on the `+`, and it will open an editor in a dialog.

 (i.e, notice how the schema config disappears behind the dialog when you click on the button)

It will be fixed automatically once `agent-merging` is merged. But it's good that you created a bug for this so that we can track progress before we release XML editor

**Comment by maxxxxxdlp** [comment]
Duplicate of #3192
